apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/supervisor/msg/SvBusResponse.java

```diff
@@ -3,19 +3,34 @@
  */
 package com.apollocurrency.aplwallet.apl.util.supervisor.msg;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
 
 /**
  * This class represents REST response in the channel
  *
  * @author alukin@gmail.com
  */
+@EqualsAndHashCode(callSuper = true)
 @JsonInclude(Include.NON_NULL)
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
 public class SvBusResponse extends SvBusMessage {
 
     /**
      * Error code, 0 or null means success
      */
-    public SvBusError error;
+    private SvBusStatus status;
```

Comment:
I think there is no reason to use private access for DTOs. It adds a lot of additional code and annotations. 
But it's question of our team rules.